test/simulation/lntest/node.go

```diff
@@ -88,6 +103,25 @@ func generateListeningPorts() (int, int, int) {
 	return p2p, rpc, rest
```

Comment:
Regarding 
```
		p2p = defaultNodePort + (3 * numActiveNodes)
		rpc = defaultClientPort + (3 * numActiveNodes)
		rest = defaultRestPort + (3 * numActiveNodes)
```
`numActiveNodes` is now redundant, since it's only applied when `numActiveNodes` is 0.

Reply:
what do you mean, actually it's only applied when numActiveNodes is different than 0?